src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -131,6 +181,17 @@ const ContentListTable = () => {
           />
         </FlexItem>
       </Flex>
+      <Flex className={classes.chips}>
+        <ContentListChips
+            searchQuery={searchQuery}
+            debouncedSearchQuery={debouncedSearchQuery}
+            setSearchQuery={setSearchQuery}
+            selectedVersions={selectedVersions}
+            setSelectedVersions={setSelectedVersions}
+            selectedArches={selectedArchitectures}
+            setSelectedArches={setSelectedArchitectures}
```

Comment:
Make these variables internal state using react context within this component.
Do not pass down setters/variables to children, use react context to get them within the child components instead. 
I did something similar in katello's bulkDeleteModal, but the idea is that you only reveal the logic needed for the parent component, in this case the setSearchQuery variable should be all you need.